resources/geocoding/en/52.txt

```diff
@@ -0,0 +1,418 @@
+52325|Acaponeta, NAY
+52326|Jalisco
+52327|Nayarit
+52328|Michoacán
+52329|Nayarit
+5233|Guadalajara, JAL
+52341|Ciudad Guzmán, JAL
+52342|Gómez Farías/Sayula, JAL
+52343|Jalisco
+52344|Mexticacan/Yahualica, JAL
+52345|Jalisco
+52346|Teocaltiche, JAL/ZAC
+52347|Jalisco
+52348|Jalisco
+52349|Jalisco
+52351|Ario de Rayón/Zamora, MICH
+52352|La Piedad, MICH
+52353|Michoacán
+52354|Michoacán
+52355|Michoacán
+52356|Tanhuato/Yurécuaro, MICH
+52357|Jalisco
+52358|Tamazula/Zapotiltic, JAL
+52371|Jalisco
+52372|Jalisco
+52373|Jalisco
+52374|Jalisco
+52375|Ameca, JAL
+52376|Jalisco
+52377|Cocula/Estipac, JAL
+52378|Jalisco
+52379|Tlajomulco de Zúñiga, JAL/MICH
+52381|Cojumatlan/San Jose de Gracia, MICH
+52382|Jalisco
+52383|Michoacán
+52384|Tala/Teuchitlan, JAL
+52385|Jalisco
+52386|Jalisco
+52387|Jalisco
+52388|Jalisco
+52389|Nayarit
+52391|Jalisco
+52392|Jamay/Ocotlán, JAL
+52393|Jalisco
+52394|Cotija de la Paz, MICH
+52395|Jalisco
+52411|Guanajuato
+52412|Guanajuato
+52413|Apaseo el Alto/Apaseo el Grande, GTO
+52414|Tequisquiapan, QRO
+52415|San Miguel Allende, GTO
+52417|Guanajuato
+52418|Dolores Hidalgo/San Diego de la Unión, GTO
+52419|Guanajuato
+52421|Guanajuato
+52422|Michoacán
+52423|Michoacán
+52424|Michoacán
+52425|Michoacán
+52426|Michoacán
+52427|San Juan del Rio, MEX/QRO
+52428|Ocampo/San Felipe, GTO
+52429|Guanajuato
+52431|Jalostotitlán/Villa Obregón, JAL
+52432|Ciudad Manuel Doblado/Romita, GTO
+52433|Zacatecas
+52434|Michoacán
+52435|Huetamo/San Lucas, MICH
+52436|Zacapu, MICH
+52437|Tlaltenango, JAL/ZAC
+52438|Michoacán
+52441|Querétaro
+52442|Querétaro
+52443|Morelia/Tarímbaro, MICH
+52444|San Luis Potosí, SLP
+52445|Moroleon, GTO
+52447|Contepec/Maravatío, MICH
+52448|Querétaro
+52449|Aguascalientes/Jesús María, AGS
+52451|Michoacán
+52452|Nuevo San Juan Parangaricutiro/Uruapan, MICH
+52453|Apatzingán, MICH
+52454|Michoacán
+52455|Michoacán
+52456|Valle de Santiago, GTO
+52457|Mezquitic, JAL/ZAC
+52458|Zacatecas
+52459|Michoacán
+52461|Guanajuato
+52462|Irapuato, GTO
+52463|Jalpa/Tabasco, ZAC
+52464|Salamanca, GTO
+52465|Aguascalientes
+52466|Guanajuato
+52467|Zacatecas
+52468|San Luis de la Paz, GTO
+52469|Buenavista de Cortés/Pénjamo, GTO
+52471|Purepero/Tlazazalca, MICH
+52472|Silao, GTO
+52473|Guanajuato, GTO
+52474|Lagos de Moreno/Paso de Cuarenta, JAL
+52475|Bajío de San José/Encarnación de Diaz, JAL
+52476|San Francisco del Rincón, GTO
+52477|León, GTO
+52478|Calera Victor Rosales, ZAC
+52481|Ciudad Valles, SLP
+52482|San Luis Potosí
+52483|San Luis Potosí
+52485|San Luis Potosí
+52486|San Luis Potosí
+52487|San Luis Potosí
+52488|San Luis Potosí
+52489|San Luis Potosí
+52492|Zacatecas
+52493|Fresnillo, ZAC
+52494|Jerez de García Salinas, ZAC
+52495|Calvillo, AGS/JAL
+52496|Zacatecas
+52498|Zacatecas
+52499|Villanueva, JAL/ZAC
+5255|Cuidad de México, CDMX
+52586|Tlalmanalco, MEX
+52587|Amecameca, MEX
+52588|Estado de Mexico
+52589|Juchitepec/Ozumba, MEX
+52591|Estado de Mexico
+52592|Estado de Mexico
+52593|Estado de Mexico
```

Comment:
This file is not in English. While it's fine to contribute the es file (and indeed that's great!) it has to be in the es folder, as es/52.txt.
Ideally you'd do both es and en.